openpype/hosts/nuke/plugins/publish/collect_nuke_instance_data.py

```diff
@@ -2,11 +2,13 @@
 import pyblish.api
 
 
-class CollectInstanceData(pyblish.api.InstancePlugin):
-    """Collect all nodes with Avalon knob."""
+class CollectNukeInstanceData(pyblish.api.InstancePlugin):
```

Comment:
Changing the class name would likely invalidate settings for `sync_workfile_version_on_families` attribute. I assume that's set from settings?